src/styles/card.css

```diff
@@ -0,0 +1,11 @@
+.card-container {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--white);
+    position: absolute;
+    top: 9rem;
+    width: 92%;
```

Comment:
If you use box-sizing, you should be able to remove the need for margin/width calculations. It just should be 100% width.

 https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

Reply:
but I actually want the card to be 92% of the tracker width. Without it, won't it have the same width as the tracker?

<img width="265" alt="image" src="https://user-images.githubusercontent.com/15824424/235458726-8cc5008f-66dc-45df-ac31-fc87b23d9f99.png">

Reply:
Why should the card care about the page padding in this case? Think about if you add another component. Let's say you want to add a full width button at the bottom that says "Confirm". You will have to add the 92% to that component also. To me, it would be better to have the page keep the padding rather than the individual elements.